components/script/dom/webidls/GPUCanvasContext.webidl

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// https://html.spec.whatwg.org/multipage/#htmlcanvaselement
+[Exposed=(Window, DedicatedWorker)]
+interface GPUCanvasContext {
+    readonly attribute (HTMLCanvasElement or OffscreenCanvas) canvas;
+};
```

Comment:
Per spec it is [[Exposed](https://webidl.spec.whatwg.org/#Exposed)=(Window, Worker), [SecureContext](https://webidl.spec.whatwg.org/#SecureContext)], but I think the root cause of error is that before this interface was hidden behind pref, but now it's not. Can we also put `Pref="dom.webgpu.enabled"` here?

Reply:
yes, pref is the root cause! thanks